modules/integration/tests-common/admin-clients/src/main/java/org/wso2/appserver/integration/common/clients/EventReceiverAdminServiceClient.java

```diff
@@ -168,4 +168,13 @@ public EventReceiverConfigurationDto getEventReceiverConfiguration(
             throw new RemoteException();
         }
     }
+
+    public boolean editEventReceiverConfiguration(String eventReceiverConfiguration, String eventReceiverName) throws RemoteException {
+        try {
+            return eventReceiverAdminServiceStub.editActiveEventReceiverConfiguration(eventReceiverConfiguration, eventReceiverName);
+        } catch (RemoteException e) {
+            log.error("RemoteException", e);
+            throw new RemoteException();
```

Comment:
@thiliA 
If you are wrapping the caught exception in a new exception, please pass the original exception as the cause for the new exception. i.e. throw new RemoteException(e);. However, in this case, since you are just logging and throwing again a remote exception, you can just log and do a 'throw e;'. Also make sure that logging the exception here does not result in the stacktrace being printed twice.